tests/models/llava/test_modeling_llava.py

```diff
@@ -80,7 +81,7 @@ def __init__(
             "initializer_range": 0.02,
             "num_labels": 3,
             "num_choices": 4,
-            "pad_token_id": 0,
+            "pad_token_id": 1,
```

Comment:
I chaneg this so that it doesn't clash with special image token, so when we artificially pad input sequence it won't get extra image placeholders than needed